core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -60,6 +60,19 @@
                       on-click-continue-button="showUpcomingCard()">
         </progress-nav>
       </div>
+      <div ng-if="isOnTerminalCard() && isCurrentCardAtEndOfTranscript()" class="conversation-skin-next-lesson-wrapper">
+        <div style="cursor: default">
+          <span class="conversation-skin-next-lesson-text">Next Lesson</span>
+          <span><[recommendedExplorationSummaries[0].title]></span>
```

Comment:
As discussed, can you change this to only show this new UI for explorations in collections and stories, but for the 'recommended Explorations' at the end of an independent exploration, show the old UI itself.

Reply:
hey @aks681 can you verify if everything is working fine?

Reply:
Not really. It works fine for collections and stories, but for explorations with recommendations at the end still the new UI is shown, but that shouldn't be the case.
You can try for yourself by creating an exploration and in the EndExploration interaction, add some exp IDs as recommended explorations, then play the exploration. At the end of the exploration, all the summary tiles corresponding to the recommended exps should be shown like before (as 3 cards at the bottom)